src/antsibull/data/ansible-release_py.j2

```diff
@@ -1,5 +1,8 @@
 # Copyright: (c) 2021, Ansible Team
 #
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2021, Ansible Team
```

Comment:
Now, some files have `# Copyright (c) ...` style copyrights, others have `# SPDX-FileCopyrightText: ...`, and this one has both. I'm fine with using both styles in different files (although I'd prefer consistency), but not the same copyright twice.

Reply:
Right now `# Copyright` is used for existing copyright statements, and `SPDX-FileCopyrightText` for files which didn't have an explicit one so far. (I personally find `SPDX-FileCopyrightText` pretty ugly, maybe we should not use it at all?)

Reply:
The duplication has been fixed in 7b50afa4dc6acddf94c21dabaf186f97fe19e530.

Reply:
> (I personally find `SPDX-FileCopyrightText` pretty ugly, maybe we should not use it at all?)

I prefer to use it, because that's what REUSE prefers (both are allowed), but I don't feel super strongly. I suggest picking one style and then documenting it in the README or standalone CONTRIBUTING doc. If we do the latter, we can also add that changelog fragments are required.